src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -116,13 +117,23 @@ struct Input
     return name;
   }
 
+  std::string getMdMeshName() const
+  {
+    using axom::utilities::string::removeSuffix;
+
+    // Remove the parent directories and file suffix
+    std::string name = axom::Path(meshFile).baseName();
+    name = removeSuffix(name, ".root");
+
+    return name;
+  }
+
   void parse(int argc, char** argv, axom::CLI::App& app)
   {
     app.add_option("-m,--mesh-file", meshFile)
       ->description(
-        "Path to computational mesh (generated by MFEMSidreDataCollection)")
-      ->check(axom::CLI::ExistingFile)
-      ->required();
+        "Path to multidomain computational mesh (generated by conduit)")
```

Comment:
Suggestion: would something like the following be more informative?
```suggestion
        "Path to multidomain computational mesh following conduit blueprint convention")
```